packages/core/src/snapshot.js

```diff
@@ -413,13 +414,23 @@ export function createSnapshotsQueue(percy) {
       let duplicate = errors?.length > 1 && errors[1].detail.includes('must be unique');
       if (duplicate) {
         if (process.env.PERCY_IGNORE_DUPLICATES !== 'true') {
-          percy.log.warn(`Ignored duplicate snapshot. ${errors[1].detail}`);
+          let errMsg = `Ignored duplicate snapshot. ${errors[1].detail}`;
+          percy.log.warn(errMsg);
+
+          await percy.suggestionsForFix(errMsg, { snapshotLevel: true, snapshotName: name });
```

Comment:
send meta here.

Reply:
Now, we are sending meta object with an error log message, I am not putting the `meta` object here as `discovery` and `snapshot` meta objects contain different values. I only want the snapshot name to display a different format in the CLI log with the snapshotName.